.github/driver.sh

```diff
@@ -0,0 +1,29 @@
+#! /bin/bash
```

Comment:
```suggestion
#!/bin/bash
```

Reply:
`shellcheck` complained about `#!/bin/bash`, so I changed it to this. I also was surprised, I'll change it back.

Reply:
Huh, I'll need to update my shellcheck then. Do you know the error code? I'd love to learn the reason for that suggestion =)

Reply:
I just rerun it, and now it didn't complain, weird. I think it suggested me that I should add a shebang with `#!/bin/bash` in the file. Not really sure though.